scripts/python/singlepss.py

```diff
@@ -217,10 +229,15 @@ def buf_get(pssName, typ, name, known):
 			else:
 				pubkey = remotekeys[name].decode("hex")
 			try:
-				feeds[name] = pss.Feed(bzzs[pssName].agent, psses[pssName].get_account(), "d" + pss.publickey_to_account(pubkey))
-				wOut(PSS_BUFPFX_DEBUG, [], "", "added feed with topic " + feeds[name].topic.encode("hex"))
+				feeds[name] = pss.Feed(bzzs[pssName].agent,
+				                       psses[pssName].get_account(),
+				                       "d" + pss.publickey_to_account(pubkey))
```

Comment:
this looks bizarre in this diff display. Look at the "d" - why so far away? How is this clearer? :)